e2e/test/scenarios/custom-column/custom-column-reproductions.cy.spec.js

```diff
@@ -416,7 +416,7 @@ describe("issue 18814", () => {
   });
 });
 
-describe.skip("issue 19744", () => {
```

Comment:
Unskipping existing repro for the issue which this entire epic is meant to fix: #19744.